_posts/2022-09-13-august-rubygems-updates.md

```diff
@@ -0,0 +1,48 @@
+---
+title: August 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in August.
+
+## RubyGems News
+
+This month in RubyGems, we released new versions of RubyGems [3.3.20](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3320--2022-08-10), [3.3.21](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3321--2022-08-24) and Bundler [2.3.20](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2320-august-10-2022), [2.3.21](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2321-august-24-2022).
+
+The following is a non-exhaustive list of other improvements included in the above releases (see the changelog for further information):
```

Comment:
This month I'd like to highlight a few specific improvements, and we can delegate the rest to the changelog.

```suggestion
The following are the main improvements shipped during this month (see the changelog for more improvements and fixes):
```

And then mention the following:

> * Full support for `musl` variants for gems, both in RubyGems and Bundler - [#4488](https://github.com/rubygems/rubygems/pull/4488), [#5852](https://github.com/rubygems/rubygems/pull/5852).
> * Implement `Bundler.settings[:only]` to install gems of the only specified groups. Longstanding feature request we finally decided to add - [#5759](https://github.com/rubygems/rubygems/pull/5759).
> * Restore previous performance of private RubyGems servers. It had gotten very slow after some correctness fixes - [#5826](https://github.com/rubygems/rubygems/pull/5826).
```

Reply:
Alright! I'll do that, thanks.